var/spack/repos/builtin/packages/hypre/package.py

```diff
@@ -100,9 +114,18 @@ def install(self, spec, prefix):
         else:
             configure_args.append('--without-MPI')
 
+        if '+openmp' in self.spec:
+            configure_args.append('--with-openmp')
+
         if '+int64' in self.spec:
             configure_args.append('--enable-bigint')
 
+        if '+mixedint' in self.spec:
+            configure_args.append('--enable-mixedint')
+
+        if '+complex' in self.spec:
+            configure_args.append('--enable-complex')
```

Comment:
Can you add an explicit `else` clause to disable these features? CMake/Autotools like to choose their defaults based on what is present on the OS, leading to non-deterministic builds.

Reply:
Yes (here ca8c577).  Thanks for the explanation.